LanguageFeatures/Augmentation-libraries/augmenting_types_A10_t02_lib.dart

```diff
@@ -49,3 +49,23 @@ augment enum E {
 // [analyzer] unspecified
 // [cfe] unspecified
 }
+
+augment extension ExtA {
+  int get foo => 42;
```

Comment:
I wouldn't expect an error here, we just can't call it outside `ExtA` using implicit resolution by referring to the name `foo` because that name is considered to denote `A.foo`.

We could still invoke it using `ExtA(A()).foo`.